src/Sentry.Hangfire/Sentry.Hangfire.csproj

```diff
@@ -14,6 +14,8 @@
   <ItemGroup>
     <ProjectReference Include="..\Sentry\Sentry.csproj" />
     <PackageReference Include="Hangfire.Core" Version="1.8.12" />
+    <!-- https://github.com/advisories/GHSA-5crp-9r3c-p9vr -->
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Do we need a dependency on N.J? If we can compile and ship the lib without it, it's up to the final app to add this override.

We should suppress the warning here

Reply:
It's a transitive dependency from `Hangfire.Core` and the version `Hangfire.Core` depends on has advisory warnings. I've tweaked this to add a reference to the same versions as Hangfire.Core and to suppress the warning:
https://github.com/getsentry/sentry-dotnet/blob/328a95eb00f7a50906e01c808511c29060392646/src/Sentry.Hangfire/Sentry.Hangfire.csproj#L19-L33

Reply:
I don't think we should add this though. We could force a consumer into a diamond dependency situation.
Can we skip the warning?